helm/alfresco-content-services/templates/secret-message-broker.yaml

```diff
@@ -9,7 +9,7 @@ type: Opaque
 data:
   {{- if .Values.activemq.enabled }}
   BROKER_URL: {{ printf "failover:(nio://%s-broker:61616)?timeout=3000&jms.useCompression=true" (include "activemq.fullname" .) | b64enc | quote }}
-  BROKER_USERNAME: {{ .Values.activemq.adminUser.username | b64enc | quote }}
+  BROKER_USERNAME: {{ .Values.activemq.adminUser.user | b64enc | quote }}
```

Comment:
we always use username as variable name otherwise here it should have been just name but not user

Reply:
generally speaking I would have preferred `username` but we have `messageBroker.user` in the main chart and is spread in different subchart so I preferred to be consistent and hopefully switch to `username` everywhere at a later time